### PR TITLE
Adding documentation for Decentralized Devstack

### PR DESCRIPTION
**Pull request by jinder1s**
This is the start of documentation for DD. I tried by best to fill in with knowledge I've gained from reading various documents, but I'm sure some of us are not on same page on things. So please comment is something is different from what you thought it would be.

I am working on further decisions in PR #159

### CODE REVIEW

**Comment by nedbat** [comment]
I hope you don't mind some copy-editing of a draft.
- s/edX/Open edX/   (there are a few that should stay edX: "edX-issued laptops").
- s/cause/because/
- s/2/two/
- put a space before opening parens
- docker should be capitalized
- You're missing articles in some places: s/These both act as central repo/These both act as a central repo/

File: oeps/oep-0005/decisions/0002-Why-Decentralized-Devstack.rst [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+Rejected Alternative
+--------------------
+
+Improve Current Devstack: TODO: give quick summary of why this was rejected
+
+
+Sources
+-------
+
+Apologizes: some of this might just be available to edX employees.
```
Comment (regisb) [inline oeps/oep-0005/decisions/0002-Why-Decentralized-Devstack.rst:54]: Is there any way we could publish those documents, or at least share them with the devstack core committers (aka: me)?

Comment (jinder1s) [reply oeps/oep-0005/decisions/0002-Why-Decentralized-Devstack.rst:54]: I'm creating this documentation in this PR to start sharing info with core committers and community. I am unsure about the sharing of docs, but I'm trying to summarize the relavant bits in the ADRs in this and future PRs.

Comment (nedbat) [reply oeps/oep-0005/decisions/0002-Why-Decentralized-Devstack.rst:54]: As an absolute minimum, "might" is just wishy-washy.  Figure out which are private and which are not.

But better: the Google Doc should be moved into the wiki and made public.  Google docs are not searchable, especially by the community.

Comment (jinder1s) [reply oeps/oep-0005/decisions/0002-Why-Decentralized-Devstack.rst:54]: The current plan is to move the docs to confluence

Comment (jinder1s) [reply oeps/oep-0005/decisions/0002-Why-Decentralized-Devstack.rst]: The links are fixed to open documents. Community members should be able to read these.

**Review by feanil — APPROVED** [review]

**Pull request merged**